Tried that already unfortunately it does not work:

```
CMake Error at CMakeLists.txt:60 (find_package):
  Could not find a configuration file for package "LibSolv" that is
  compatible with requested version "0.7.21".

  The following configuration files were considered but not accepted:

    /usr/share/cmake/libsolv/LibSolvConfig.cmake, version: unknown
```

If the path appended doesn't exist, it's simply ignored, it won't cause any failures. Do you foresee any issue with just adding it?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,6 +17,9 @@ set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
 if(${CMAKE_VERSION} VERSION_LESS 3)
     set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH};${CMAKE_SOURCE_DIR}/cmake/modules-cmake-2)
 endif()
+# On Ubuntu et al. CMake is configured differently and needs to be told where to find libsolv,
+# otherwise the build fails
+LIST (APPEND CMAKE_MODULE_PATH "${CMAKE_INSTALL_PREFIX}/share/cmake/libsolv/")
 
 
 # print exact name and version of the compiled project

